### ISSUE TITLE
CTI by Short et al. has no effect on the maximum value

### ISSUE THREAD

**Issue by SaraRegibo** [IvS-KULeuven/PlatoSim3#403]
This issue was notice when testing the CTI (Short et al. 2013) for the verification & validation plan.

Use the same script as in #402 but now enable CTI (Short et al. 2013), like this:

````
# Switch on CTI (simple implementation)

sim["CCD/IncludeCTIeffects"] = "yes"
sim["CCD/CTI/Model"] = "Short2013"
````

In this case the maximum flux in the simulated columns does not change, although CTI has been switched on (this can be checked via the printouts, similar as in #402 ).  I expect the maximum flux in the column to be lower when the source was inserted further away from the readout register (as more parallel charge transfers are needed for the charges in these rows to reach the readout register).

**Comment by SaraRegibo** [comment]
The notebook demonstrating the problem can be found in my personal PlatoSim repository in the `validation-verification` branch.  It's the `Charge Transfer Inefficiency` notebook.

**Comment by SaraRegibo** [comment]
Fixed in #463.
